modules/web/src/app/project-overview/component.ts

```diff
@@ -141,6 +142,9 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
   }
 
   isEditEnabled(): boolean {
+    if (this.restrictProjectModification) {
+      return false;
```

Comment:
Editing should be allowed for admins.
```suggestion
      return this.currentUser.isAdmin;
```

Reply:
Updated